docs/rules/no-empty-glimmer-component-classes.md

```diff
@@ -0,0 +1,34 @@
+# no-empty-glimmer-component-classes
+
+This rule will catch and prevent the use of empty backing classes for Glimmer components.
+
+## Rule Details
+
+This rule aims to disallow the use of empty backing classes for Glimmer components. Template-only Glimmer components where there is no backing class are much faster and lighter-weight than Glimmer components with backing classes, which are much lighter-weight than Ember components. Therefore, you should only have a backing class for a Glimmer component when absolutely necessary. Otherwise, remove the backing class entirely until it is actually needed.
```

Comment:
Is there any documentation we could link to which confirms this reasoning?

Reply:
Probably the RFC, I think there is something wrong with the api docs (I tried linking to them but the templateOnly sidebar item doesn't link to anything).